.travis/klee.sh

```diff
@@ -208,6 +208,20 @@ else
 
   set +e # We want to let all the tests run before we exit
   lit -v .
+
+  # If metaSMT is the only solver, then rerun tests with non-default metaSMT backends
+  if [ "X${SOLVERS}" == "XmetaSMT" ]; then
+    if [ "X${METASMT_DEFAULT}" != "Xbtor" ]; then
+      lit -v --param klee_opts=-metasmt-backend=btor --param kleaver_opts=-metasmt-backend=btor .
+    fi
+    if [ "X${METASMT_DEFAULT}" != "Xstp" ]; then  
+      lit -v --param klee_opts=-metasmt-backend=stp --param kleaver_opts=-metasmt-backend=stp .
+    fi
+    if [ "X${METASMT_DEFAULT}" != "Xz3" ]; then
+      lit -v --param klee_opts=-metasmt-backend=z3 --param kleaver_opts=-metasmt-backend=z3 .
+    fi
+  fi
+
```

Comment:
This part is wrong because we do `set +e` before running all the tests. This allows us to run all the tests and then we check the value of `$RETURN` at the end. i.e.

```
 if [ "${RETURN}" != "00" ]; then
     echo "Running tests failed"
     exit 1
 fi
```

Now that the number of tests we run is variable is might be better (and easier) if we just remove `set +e` and fail fast (i.e. We stop running at the first failing command). Otherwise the logic for checking at the return codes would get a little complicated.